client/state/data-layer/wpcom/timezones/index.js

```diff
@@ -43,21 +43,24 @@ export const fromApi = ( { manual_utc_offsets, timezones, timezones_by_continent
 /*
  * Start a request to WordPress.com server to get the timezones data
  */
-export const fetchTimezones = ( { dispatch }, action ) =>
-	dispatch(
-		http(
-			{
-				method: 'GET',
-				path: '/timezones',
-				apiNamespace: 'wpcom/v2',
-			},
-			action
-		)
+export const fetchTimezones = action =>
+	http(
```

Comment:
Changing these handlers to return the action instead of dispatching it will break the current tests.